.github/workflows/on_tag_deploy_to_production.yaml

```diff
@@ -58,13 +58,29 @@ jobs:
         run: |
           scp -o StrictHostKeyChecking=no .env ${{ secrets.PROD_EC2_USER }}@${{ secrets.PROD_EC2_HOST }}:/home/${{ secrets.PROD_EC2_USER }}/.env
 
-      - name: SSH into EC2 and run Docker image
+      - name: SSH into EC2, install dependencies and run docker image
         run: |
           ssh -o ServerAliveInterval=60 -o ServerAliveCountMax=60 -o StrictHostKeyChecking=no ${{ secrets.PROD_EC2_USER }}@${{ secrets.PROD_EC2_HOST }} << 'EOF1'
+            # Navigate to your project directory (update if necessary)
```

Comment:
We're actually doing this already on docker. If you look at the docker file, we have pip install there.  There error you're seeing is actually from the previous deployment where we install packages manually. The real logs you should be seeing is the one from the docker logs.